db_stress_tool/db_stress_shared_state.h

```diff
@@ -58,7 +58,7 @@ class SharedState {
   static bool ignore_read_error;
 #endif // ROCKSDB_SUPPORT_THREAD_LOCAL
 
-  SharedState(Env* env, StressTest* stress_test)
+  SharedState(Env* /*env*/, StressTest* stress_test)
```

Comment:
Can we give a flag to decide whether the customized env will be used or not?

Reply:
We could do that, but I wonder if there's any advantage in storing the expected values file using the customized env? Why not always use the default env? Its unlikely we'll want to do fault injection or anything like that on the expected values file.